src/epcpm/parameterstointerface.py

```diff
@@ -460,11 +463,19 @@ def gen(self):
                 sunspec_type,
             ]
 
-            sunspec_getter = "_".join(str(x) for x in getter_setter_list + ["getter"])
+            indexed_str = ""
+            if parameter.getter_index is not None:
+                indexed_str = "indexed_"
+
+            sunspec_getter = "_".join(str(x) for x in getter_setter_list + [indexed_str + "getter"])
```

Comment:
What about `indexed` maybe being another option in the list of `variable`, `function`, and `constant`?